Makefile

```
@@ -52,7 +52,10 @@ installdocker:
 testdocker:
 	docker-compose run --rm php sh -c 'vendor/bin/phpunit'
 
-.PHONY: all check-style fix-style install test clean clean_all up cli installdocker migrate testdocker
+efs: # Everything From Scratch
+	make clean_all; make up; make migrate; make cli; echo "---- ./vendor/bin/phpunit ----"
```

```suggestion
efs: clean_all up migrate # Everything From Scratch
```

instead of running make inside of make, you can define dependencies, `efs` depends on `clean_all`, `up`, `migrate` and it will run these in that order before running the commands defined under efs.

I'd skip `make cli` in this case, you can use it like

```
make efs cli
```

to have the same effect.